.atomist/tslint.json

```diff
@@ -5,10 +5,14 @@
     ],
     "jsRules": {},
     "rules": {
-        "interface-name": [true, "never-prefix"],
+        "interface-name": [
+            true,
+            "never-prefix"
+        ],
         "max-classes-per-file": false,
         "object-literal-sort-keys": false,
-        "no-console": false
+        "no-console": false,
+        "radix": false
```

Comment:
Do we really want to disable this rule?
https://palantir.github.io/tslint/rules/radix/